tests/robot-tests/tests/admin_and_public/bau/prelease_visibility.robot

```diff
@@ -0,0 +1,223 @@
+    user clicks button  Save guidance
+
+Go to "Sign off" page
+    [Tags]  HappyPath
+    user clicks link   Sign off
+    user waits until h2 is visible  Sign off
+    user waits until page contains button  Edit release status
+
+Approve release and wait for it to be Scheduled
+    [Tags]  HappyPath
+    ${day}=         get current datetime  %-d  2
+    ${month}=       get current datetime  %-m  2
+    ${month_word}=  get current datetime  %B  2
+    ${year}=        get current datetime  %Y  2
+
+    user clicks button  Edit release status
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote     Approved by prerelease UI tests
+    user waits until page contains element   xpath://label[text()="On a specific date"]/../input
+    user clicks radio   On a specific date
+    user waits until page contains   Publish date
+    user enters text into element  id:releaseStatusForm-publishScheduled-day    ${day}
+    user enters text into element  id:releaseStatusForm-publishScheduled-month  ${month}
+    user enters text into element  id:releaseStatusForm-publishScheduled-year   ${year}
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-month   1
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-year    2001
+    user clicks button   Update status
+
+    user waits until h2 is visible  Sign off
+    user checks summary list contains  Current status  Approved
+    user checks summary list contains  Scheduled release  ${day} ${month_word} ${year}
+    user checks summary list contains  Next release expected  January 2001
+    user waits for release process status to be  Scheduled  90
+
+Navigate to prerelease page
+    [Tags]  HappyPath
+    ${current_url}=  get location
+    ${RELEASE_URL}=  remove substring from right of string  ${current_url}  /status
+    set suite variable  ${RELEASE_URL}
+    user goes to url   ${RELEASE_URL}/prerelease
+
+Validate prerelease has not started
+    [Tags]  HappyPath
+    user waits until h1 is visible  Pre-release access is not yet available
+    user checks breadcrumb count should be   2
+    user checks nth breadcrumb contains   1    Home
+    user checks nth breadcrumb contains   2    Pre-release access
+
+    ${day}=         get current datetime  %d   1
+    ${month}=       get current datetime  %m   1
+    ${year}=        get current datetime  %Y   1
+    ${time_start}=  format uk to local datetime  ${year}-${month}-${day}T00:00:00  %-d %B %Y at %H:%M
+    ${time_end}=    format uk to local datetime  ${year}-${month}-${day}T23:59:00  %-d %B %Y at %H:%M
+    user checks page contains   Pre-release access will be available from ${time_start} until ${time_end}.
+
+Go to prerelease access page
+    [Tags]  HappyPath
+    user goes to url   ${RELEASE_URL}/prerelease-access
+    user waits until h2 is visible  Manage pre-release user access
+
+Go to Public release page and ensure release isn't visible
```

Comment:
I think this test case can be removed as well. The `Go to release URL and check release isn't visible` validates the `${PUBLIC_RELEASE_LINK}` page.